resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -224,7 +225,9 @@ cloudwatch.logs.opening_in_editor=Opening Stream {0} in editor
 cloudwatch.logs.query=Open Query Editor
 cloudwatch.logs.query.form.ok_Button=Apply
 cloudwatch.logs.query_editor_title=Query Log Groups
+cloudwatch.logs.query_name=Query name must be entered
 cloudwatch.logs.save_action=Save to a File
+cloudwatch.logs.save_query=Save Query
 cloudwatch.logs.saving_to_disk=Saving Stream {0} to disk
 cloudwatch.logs.saving_to_disk_failed=Saving Stream {0} failed
 cloudwatch.logs.saving_to_disk_succeeded=Finished saving Log Stream {0} to {1}
```

Comment:
Can you add a gif of the functionality to the PR?

Reply:
I don't think there's much to show with a gif in this PR considering there's no listing functionality.